de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/concepts/util/ActiveConceptHelper.java

```diff
@@ -188,6 +188,19 @@ private static Concept getConceptForEObject(EObject conceptType) {
 		return (currentType instanceof Concept) ? (Concept) currentType : null;
 	}
 	
+	/**
+	 * Method to get the concept name to display in concept selection dialog
+	 * @param concept the concept which name details are needed
+	 * @return the display name plus FQN and version or the FQN and version when there is no display name
+	 */
+	public static String getConceptName(Concept concept) {
+		if (concept.getDisplayName() == null) {
+			return concept.getName() + " [" + concept.getVersion() + "]";

```

Comment:
I think we can make something like `String formattedVersion = [" + concept.getVersion() + "]"` so we dont duplicate this logic.